mlrun/runtimes/nuclio/function.py

```diff
@@ -462,7 +462,7 @@ def add_v3io_stream_trigger(
         name="stream",
         group="serving",
         seek_to="earliest",
-        shards=1,
+        shards=None,
```

Comment:
Maybe we should require this parameter, or try to check it for the user (i.e. `DescribeStream`), because as I remember, the design is to have one replica per shard, because V3IO streams don't support rebalancing.